### ISSUE TITLE
error occur when training my own data

### ISSUE THREAD

**Issue by OPPOA113** [Stick-To/CenterNet-tensorflow#3]
Nice works.
something like image's wrong occur when i training,is right?
![snipaste_20190528_160320](https://user-images.githubusercontent.com/4974108/58461554-8898bc80-8162-11e9-9399-067c420c731b.jpg)
and i want to know how to filter such mistake image when make tfrecord datasets.
thankyou

**Comment by whatlas** [comment]
i got some errors like this, but on my dataset the training code cannot even go to the "iters xxx/xxxx Loss: xxxx"line , it just print "---------------epoch 0 --------------" and crashes at the very benginning of the training ,any help?

**Comment by OPPOA113** [comment]
i have no idea of your error. about mine was cause by the wrong images' format, after i filter such bad images, i run well.
